research/specfiles/hosting-sources/index.md

```diff
@@ -0,0 +1,372 @@
+---
+title: Hosting sources for Fedora
+authors: mfocko
+---
+
+:::tip Context
+
+tl;dr As long as Packit is deployed on the MP+, onboarding of new users is
+limited by the firewall rules that need to be explicitly allowed. Additionally
+firewall is mostly blocking `sync-release` actions specifically.
+
+This is outcome of [this](https://github.com/packit/packit-service/issues/2390)
+issue.
+
+:::
+
+All `SourceX` fields of the specfiles have been initially scraped by the @msuchy.
+This “research” provides the scripts that have been used to process the scraped
+sources.
+
+### Domains with ≥ 10 occurrences
```

Comment:
so, what do you think will be the reasonable threshold for us to ask for the firewall adjustments?

Reply:
Added some of the bigger domains, it appears that with dependencies it's easy to get _≥ 10_. Additionally we don't have many blocked packages on the firewall, so I would consider only the really big hosts (like forges).